api/controllers/EquationController.js

```diff
@@ -164,7 +164,11 @@ module.exports = {
 				if (req.wantsJSON) {
 					return res.json(equation); 
 				} else {
-					return res.redirect("#/equation/" + equation.id);
+                    if (equation.components.length > 0) {
```

Comment:
Is this indentation correct?

Reply:
The indentation is correct if you use 4 spaces tabs.  Problems of mixing spaces and tabs :(